torch/distributions/von_mises.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+import torch
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions.utils import broadcast_all

```

Comment:
You need to import lazy_property as well, and I think that should fix the failures.

Reply:
Thanks!